File: hw/marquez/u2/HW18/HW18ZooInPythonJavaScript/zoo/ec.edu.espe.zoo.model/Tiger.js
const { Feline } = require('./Feline.js'); // Importar clase Feline desde otro archivo

class Tiger extends Feline {
    constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    Feed(food) {
        throw new Error("Not supported yet.");
    }
}

module.exports = { Tiger };

File: hw/travez/u2/HW18ZooInPython/ec_edu_espe_zoo_model/Tiger.py
from datetime import datetime
from Feline import Feline

# Ismael Travez, The Java Bandits, DCCO-ESPE

class Tiger(Feline):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        print("Feeding to a lion")
